javacord-api/src/main/java/org/javacord/api/util/DiscordRegexPattern.java

```diff
@@ -85,6 +85,7 @@
                             + "(?i)                             # ignore case \n"
                             + "(?:https?+://)?+                 # 'https://' or 'http://' or '' \n"
                             + "discord(?:app)?+\\.com/channels/ # 'discord(app).com/channels/' \n"
+                            + "(?:(?:canary|ptb)\\.)?+          # 'canary.' or 'ptb.'\n"
```

Comment:
In order to match the canary and ptb links this line should be moved between the **https** and **discord**
```suggestion
                            + "(?:https?+://)?+                 # 'https://' or 'http://' or '' \n"
                            + "(?:(?:canary|ptb)\\.)?+          # 'canary.' or 'ptb.'\n"
                            + "discord(?:app)?+\\.com/channels/ # 'discord(app).com/channels/' \n"
```

Reply:
thx. I first changed it somewhere else and then copied in the wrong line. fixed it